numpy/f2py/crackfortran.py

```diff
@@ -1494,9 +1494,11 @@ def analyzeline(m, case, line):
         f = 0
         bn = ''
         ol = ''
+        nslash = 0
         for c in line:
-            if c == '/':
+            if c == '/' and nslash < 2:
```

Comment:
This is a good pragmatic approach, could you add a comment explaining this is for counting the common block instances? 

Actually it would be best though to regex match the common block section so only `common/blah/` gets skipped instead of relying on the count.

Reply:
Good idea. It turned into a one-liner using `re.split`.